src/main/java/io/github/jmh07/scoutingeyes/ScoutingEyesTabCompletion.java

```diff
@@ -0,0 +1,41 @@
+package io.github.jmh07.scoutingeyes;
+
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.TabCompleter;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ScoutingEyesTabCompletion implements TabCompleter {
+    @Override
+    public List<String> onTabComplete(CommandSender commandSender, Command command, String s, String[] args) {
+
+        if(args.length == 1){
+            List<String> option= new ArrayList<>();
+            option.add("basic");
+            option.add("omega");
+            return  option;
+        }
+        if(args.length == 2){
+            List<String> number= new ArrayList<>();
+            number.add("1");
+            number.add("64");
+            return number;
```

Comment:
Recommend checking if args[1] is greater than 64 and if it is to return an error message instead to tell the user that it is an invalid amount.

Reply:
I don't see why set a limit. It could be used to fill up a chest or something else. I don't think there is a need to put such limits and I would prefer allow the user to be responsable.